config/tailwind/v4.x.x/css-config/src/input.css

```diff
@@ -0,0 +1,284 @@
+/* Import Tailwind CSS */
+@import "tailwindcss";
+
+/* Import GCDS tokens */
+@import "../node_modules/@cdssnc/gcds-tokens/build/web/css/tokens.css";
```

Comment:
Same import clean up comment as above.

```suggestion
@import "@cdssnc/gcds-tokens/build/web/css/tokens.css";
```